packages/vue/src/components/list/SingleDropdownList.jsx

```diff
@@ -257,7 +260,7 @@ const SingleDropdownList = {
 
 		handleChange(item) {
```

Comment:
Update it too.

Reply:
we don't need here, it directly accepts `item` as a param rather than `event`